integration_tests/CMakeLists.txt

```diff
@@ -360,6 +363,13 @@ macro(RUN)
         list(REMOVE_ITEM RUN_LABELS cpython cpython_sym) # remove cpython, cpython_sym, from --fast test
         RUN_UTIL(RUN_FAIL RUN_NAME RUN_FILE_NAME RUN_LABELS RUN_EXTRAFILES RUN_NOMOD RUN_EXTRA_ARGS RUN_COPY_TO_BIN)
     endif()
+    
+    # if (llvm IN_LIST RUN_LABELS)
+    #     unset(RUN_EXTRA_ARGS)
+    #     set(RUN_EXTRA_ARGS --jit)
+    #     set(RUN_NAME "${RUN_NAME}_JIT")
+    #     RUN_UTIL(RUN_FAIL RUN_NAME RUN_FILE_NAME RUN_LABELS RUN_EXTRAFILES RUN_NOMOD RUN_EXTRA_ARGS RUN_COPY_TO_BIN)
```

Comment:
I didn't intend to push this. I was testing stuff out in my local computer, I should have removed this before pushing. Sorry. Will remove it in the next commit.